sql/moz-fx-data-shared-prod/firefox_desktop_derived/newtab_clients_daily_aggregates_v2/query.sql

```diff
@@ -0,0 +1,84 @@
+SELECT
+  submission_date,
+  app_version,
+  os,
+  channel,
+  locale,
+  browser_version,
+  country,
+  newtab_homepage_category,
+  newtab_newtab_category,
+  organic_content_enabled,
+  sponsored_ads_enabled,
```

Comment:
we are being inconsistent with the naming between `sponsored_ads` and `sponsored_content`. Let's go with `sponsored_content` for now. How easy/difficult is it to change the column naming? I am to still track down the best way to differentiate organic/sponsored stories that makes sense to both Ads/HNT folks.

`sponsored_ads` appear in lines 12, 81, 16, 92, 320 - 321. This should be changed to `sponsored_content`, at least for now.

Reply:
Just chiming in to say `sponsored_content` is fine on our end